graph/appender/dead_node.go

```diff
@@ -103,16 +87,12 @@ func applyDeadNodes(trafficMap graph.TrafficMap, wkSvc business.WorkloadService,
 				continue
 			}
 
-			// Remove if backing deployment is not defined, flag if there are no pods
-			// Note that in the future a workload could feasibly be back by something
-			// other than a deployment; we may need to query the workload name againts
-			// various possibly entities.
-			workload, err := wkSvc.GetWorkload(n.Namespace, n.Workload, false)
-			if err != nil || workload == nil {
+			// Remove if backing workload  is not defined, flag if there are no pods
```

Comment:
`...workload  is...` TWO SPACES! OCD ALERT!

Reply:
fine.